scripts/models/garageModel.js

```diff
@@ -0,0 +1,74 @@
+(function(module) {
+
+  var garage = {};
+
+  function ParkedCar(data, splitCar, car) {
+    this.make = splitCar[0];
+    this.model = splitCar[1];
+    this.year = splitCar[2];
+    this.id = splitCar[3];
+    this.content = car.replace(/\//g, '_');
+    this.dataContent = car;
+    this.maintenance = data.actionHolder;
+  }
+
+  garage.allParkedCars = [];
+
+  var renderMaintenace = function(carData) {
+    // console.log(carData);
```

Comment:
As part of any final submission, you should be removing comments and debugging code. Not a big deal here, but also not something you want in any production environment unless absolutely necessary.